openedx/features/journals/templates/journals/bundle_about.html

```diff
@@ -0,0 +1,162 @@
+## mako
+<%page expression_filter="h"/>
+
+<%inherit file="/main.html" />
+<%!
+from datetime import datetime
+
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+from mako import exceptions
+
+from urlparse import urljoin
+
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<%namespace name='static' file='../static_content.html'/>
+
+## Override the default styles_version to use Bootstrap
+<%! main_css = "css/bootstrap/lms-main.css" %>
+
+
+<%
+%>
+
+<%block name="js_extra">
+  <script src="${static.url('js/leanModal.js')}"></script>
+  <script src="${static.url('js/program_marketing.js')}"></script>
+</%block>
+
+<%block name="pagetitle">${bundle['title']}</%block>
+<%block name="marketing_hero">
+  <%
+    banner_image = bundle.get('banner_image', {}).get('large', {}).get('url', '')
+    price_format = '{0:.0f}' if bundle['pricing_data']['total_incl_tax'].is_integer() else '{0:.2f}'
+  %>
+  <div id="program-details-hero">
+    <div class="main-banner"
+      style="background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), url(${banner_image});">
+      <div class="container" >
+        <div class="row">
+          <div class="col col-12 col-md-8">
+            <div>
+              <h1 class="program_title">${bundle.get('title', '')}</h1>
+            </div>
+            <div>
+              ## Note: Weird formatting to fix the inline spacing issue.
+              <a href="${bundle['pricing_data']['purchase_url']}" class="btn btn-success">
+              <span>${_('Purchase the Bundle (')}</span
+              % if bundle['pricing_data']['is_discounted']:
+                ><span aria-label="${_('Original Price')}" class="original-price"
+                >${Text(_('${oldPrice}')).format(
+                    oldPrice=price_format.format(bundle['pricing_data']['total_incl_tax_excl_discounts'])
+                  )}</span
+                ><span aria-label="${_('Discounted Price')}" class="discount">
+                    ${Text(_('${newPrice}')).format(
+                      newPrice=price_format.format(bundle['pricing_data']['total_incl_tax']),
+                    )}
+                </span
+                ><span class="savings">
+                  ${Text(_('{currency})')).format(
```

Comment:
No need to wrap a standalone variable with `gettext`

Reply:
fixed